src/cljs/main/broadfcui/header.cljs

```diff
@@ -41,7 +41,8 @@
    {:label [:div {:style {:borderRadius 2
                           :backgroundColor (:background-light style/colors)
                           :color "#000" :textDecoration "none"
-                          :padding "0.5rem" :border style/standard-line}}
+                          :padding "0.5rem" :border style/standard-line}
+                  :data-test-id "accountDropdown"}
```

Comment:
I have reservations about converting keys, but that's a separate concern. I will kebab-case this one.